lib/tasks/scsb.rake

```diff
@@ -36,6 +36,21 @@ namespace :scsb do
     IndexFunctions.process_scsb_dumps([dump], ENV['SET_URL'])
   end
 
+  desc "Adds a local PARTNER_RECAP file to the database"
+  task add_local_partner_recap_dump: :environment do
+    abort "usage: FILE=full-file-path.xml.gz rake add_local_partner_recap_dump" unless ENV['FILE']
+    ev = Event.new(success: true)
+    ev.save
+
+    dump_type_id = DumpType.where(constant: "PARTNER_RECAP").first.id
+    dump = Dump.new(event_id: ev.id, dump_type_id: dump_type_id)
```

Comment:
can we add constant: DUMP_TYPE so that we can pass any constant we want when we run the rake task?